### PR TITLE
`fn uncompress`: cleanup

### PR DESCRIPTION
**Pull request by folkertdev**

### CODE REVIEW

File: bzip2.rs [hunk 1]
```diff
@@ -2050,13 +2050,11 @@ unsafe fn main_0(program_path: &Path) -> IntNative {
...
     signal(
-        11 as libc::c_int,
-        mySIGSEGVorSIGBUScatcher as unsafe fn(IntNative) as usize,
-    );
-    signal(
-        7 as libc::c_int,
+        11,
         mySIGSEGVorSIGBUScatcher as unsafe fn(IntNative) as usize,
     );
+    signal(7, mySIGSEGVorSIGBUScatcher as unsafe fn(IntNative) as usize);
```
Comment (bjorn3) [inline bzip2.rs:2098]: These should be `extern "C"`.

**Review by bjorn3 — COMMENTED** [review]
The "fn uncompress: make magicNumberOK a bool" commit should be dropped as it is empty.

File: bzip2.rs (hunk 1 above)
Comment (folkertdev) [reply bzip2.rs:2098]: yes, though we might be able to remove these entirely ultimately?

**Comment by folkertdev** [comment]
the empty commit has been removed, and I cleaned up the signal handlers a bit too.

**Review by bjorn3 — APPROVED** [review]

**Pull request merged**